.github/workflows/ci.yml

```
@@ -21,14 +21,34 @@ jobs:
+      - name: Install diff-root
+        run: |
+          go get -u go-hep.org/x/hep/groot/cmd/root-diff
+
+      - name: Test simulation output
+        run: |
+          if ! diff-root -t tests/refs/ODD_sim.root tests/run/ODD_sim.root; then
+            echo "Output differs from reference!" >&2
+            exit 1
+          else
+            echo "Output matches reference."
+          fi
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)